__mocks__/@grafana/ui.tsx

```diff
@@ -1,6 +1,8 @@
 // @ts-ignore
 window.__react_router_build__ = undefined;
 
+import { TextEncoder } from 'util';
+global.TextEncoder = TextEncoder;
```

Comment:
I don't like to add this here, but without it all tests will fail even though it's also present in `jest.setup.tsx`
![image](https://github.com/grafana/synthetic-monitoring-app/assets/6271380/392cc58f-c91b-4116-9892-bf33288e5cfc)

Should I remove it from `jest.setup.tsx` and just leave it here? It doesn't seem like the best place though. The odd thing is this was not needed when including the workaround in `setupFilesAfterEnv` in a separate file (https://github.com/grafana/synthetic-monitoring-app/pull/815/commits/5a6e7b2174fd5905d480151bf4fcfafc6968f861)

Reply:
In the end I removed it from here as importing the `encoder-workaround` file directly from `jest-setup.tsx` worked just fine.